whenthen-web/src/components/AlertComponent/index.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect, useState, useRef } from 'react';
+import { Overlay, Button } from 'react-bootstrap';
+import useRootData from '../../hooks/useRootData';
+import stylesDesktopDefault from './DesktopDefault.module.scss';
+
+const AlertComponent = () => {
+  const {
+    screenClass,
+    alertModalVisibility,
+    alertModalContent,
+    setAlert,
+    removeAlert,
+  } = useRootData(({ appStore }) => ({
+    screenClass: appStore.screenClass.get(),
+    alertModalVisibility: appStore.alertModalVisibility.get(),
+    alertModalContent: appStore.alertModalContent.get(),
+    setAlert: appStore.setAlert,
+    removeAlert: appStore.removeAlert,
+  }));
+
+  const isDesktop = screenClass === 'xl';
+
+  const styles = isDesktop ? stylesDesktopDefault : stylesDesktopDefault;
+  const alertRef = useRef<HTMLDivElement>(null);
+  const overlayTarget = useRef(null);
+
+  useEffect(() => {
+    const onClickEvent = (event: MouseEvent) => {
+      const target = event.target as HTMLDivElement;
+      if (target === alertRef.current && alertModalVisibility) {
+        removeAlert();
+      }
+    };
+
+    document.addEventListener('click', onClickEvent);
+
+    return () => {
+      document.removeEventListener('click', onClickEvent);
+    };
+  });
+
+  return (
+    <Overlay target={overlayTarget.current} show={alertModalVisibility}>
+      {() => (
+        <div ref={alertRef} className={styles.mainBlock}>
+          <div className={styles.alertBlock}>
+            <h4>{alertModalContent}</h4>
```

Comment:
Is this the best you can do?
Align text to the center (not only horizontally but also vertically)
Add an Enum type that indicates the type of alert box (such as Warning, Information, Confirmation, ...), each different alert type would have a different layout and icons
When you make new features, google it before you start working on it and benchmark & reference those search results (you can google with `react alert popup` and you can get a bunch of images of other's designs)

Reply:
OMG H4 TAG & PLZ fix align issues

Reply:
The alignment issues will be fixed in the next commit.